Doc in_tail permission

brsolomon-deloitte opened the pull request:
Related:

- https://github.com/fluent/fluentd/issues/3525
- https://github.com/fluent/fluentd/pull/3478

cosmo0920 commented on input/tail.md:
```diff
@@ -417,6 +417,13 @@ If you see this message:
 
 It means that `fluentd` does not have read permission for `/path/to/file`. Check your fluentd and target files permission.
 
+**Note**: When `td-agent` is launched by systemd, the user of the `td-agent` process is the `td-agent` user.
```
_by default_ is needed?

the user of the `td-agent` process is the `td-agent` user _by default_.

brsolomon-deloitte replied:
Clarified in e6d9693

cosmo0920 commented on parser/README.md:
````diff
@@ -41,6 +41,11 @@ Here is an example to read Nginx access logs using `in_tail` and `parser_nginx`:
 </source>
 ```
 
+**Note**: When `td-agent` is launched by systemd, the user of the `td-agent` process is the `td-agent` user.
````
ditto.

brsolomon-deloitte replied:
Clarified in e6d9693

kenhys commented on input/tail.md:
```diff
@@ -417,6 +417,13 @@ If you see this message:
+this case `td-agent` must be added to the `adm` group.
+
+A bug exists in Fluentd 1.13.x where it may suppress warning logs about unreadable files. (See Fluentd PR [#3478](https://github.com/fluent/fluentd/pull/3478).)
+
```
Adding a link to https://docs.fluentd.org/deployment/linux-capability#capability-handling-on-in_tail may be useful.

brsolomon-deloitte replied:
Well I just learned something new about fluentd.

Added in 1668af1.

cosmo0920 reviewed (changes requested):
commit https://github.com/fluent/fluentd-docs-gitbook/pull/370/commits/e6d969369357585ee83590b8d43836c1fb2d40b3 is not included Signed-off-by line. Could you add it there?

brsolomon-deloitte committed:
Better document in_tail's need for read permission

- Clarify that td-agent is *default* user of process
- Describe that cap_dac_read_search may also be used
  in addition to usermod

Related:

- https://github.com/fluent/fluentd/issues/3525
- https://github.com/fluent/fluentd/pull/3478

Signed-off-by: Brad Solomon <81818815+brsolomon-deloitte@users.noreply.github.com>

brsolomon-deloitte commented:
@cosmo0920 done, squashed commits and signed.

cosmo0920 reviewed (approved):
Great. Thanks!
